src/Backend/Core/Engine/Url.php

```diff
@@ -253,9 +253,9 @@ private function getInterfaceLanguage(): string
             return Authentication::getUser()->getSetting('interface_language', $default);
         }
 
-        if (CommonCookie::exists('interface_language')) {
+        if ($this->getContainer()->get('fork.cookie')->has('interface_language')) {
```

Comment:
well i'd say check your configuration, `$this->get('fork.cookie')` works just fine here